conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/FeignJaxRsClientBuilder.java

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-package com.palantir.remoting3.jaxrs;
+package com.palantir.conjure.java.client.jaxrs;
```

Comment:
what are differences between packages `com.palantir.conjure.java.client` and `com.palantir.conjure.java.clients`?

Reply:
Mistype, fixed